### PR TITLE
[FIX] *: correct typos and bad English

### PR DESCRIPTION
**Pull request by mart-e**
Courtesy of translators
Reexport .pot of modified modules

### CODE REVIEW

**Comment by mart-e** [comment]
@robodoo r+

**Comment by tde-banana-odoo** [comment]
Are you sure for forums ?

**Comment by mart-e** [comment]
@robodoo r+ again !
@tde-banana-odoo maybe we should change the key instead of the label  :confused:

**Comment by tde-banana-odoo** [comment]
@mart-e Would require migration script. Tant pis.

File: addons/website_slides/models/slide_channel.py [hunk 1]
```diff
@@ -121,10 +121,10 @@ def _get_default_enroll_msg(self):
         string="Promoted Content", default='latest', required=False,
         help='Depending the promote strategy, a slide will appear on the top of the course\'s page :\n'
              ' * Latest Published : the slide created last.\n'
-             ' * Most Voted : the slide which have to most vote.\n'
+             ' * Most Voted : the slide which have to most votes.\n'
```
Comment (Feyensv) [inline addons/website_slides/models/slide_channel.py:124]: * which **has the** most votes
or
* the **slides** which have **the** most votes

Comment (mart-e) [reply addons/website_slides/models/slide_channel.py]: good catch, thanks

**Comment by mart-e** [comment]
@robodoo r+

**Pull request closed without merging**